frontend/src/app/Minigames.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+    // Note: I will remove the uneccessary comments/imports when I am done! 
+    // eslint-disable-next-line
+    Tooltip, Whisper, Alert, Dropdown, FlexboxGrid, Icon, IconButton
+} from "rsuite";
+
+// Primarily pulled from app header
+
+class MiniGames extends React.Component {
+    // constructor(props) {
+    //     super(props);
+    //     // TODO: make backend endpoint to return the filter categories
+    //     // const filterCategories = [
+    //     //     "Page Content",
+    //     //     "URL",
+    //     //     "Title",
+    //     //     "Next Connections",
+    //     //     "Previous Connections",
+    //     //     "Highlights",
+    //     //     "Notes"
+    //     // ]
+    //     // this.state = {
+    //     //     indeterminate: false,
+    //     //     checkAll: true,
+    //     //     value: filterCategories,
+    //     //     filterCategories: filterCategories
+    //     // };
+    // }
+
+    render() {
+        return (
+                <Whisper preventOverflow trigger="hover" speaker={<Tooltip>MiniGames</Tooltip>}
+                         placement="topEnd">
+                <IconButton id="projects-sidebar-btn" appearance="primary" icon={<Icon icon="bolt"/>} circle
+                    size="lg" onClick={() => Alert.warning("Feature coming soon...")} 
+                    style={{
+                        top: '50%'
+                    }}/>
+                </Whisper>
```

Comment:
I checked out your branch and tested a few ways to make the button look a little nicer (and remove some unnecessary things). Note that I'm removing the id you added to the button, since IDs have to be unique to DOM elements, and many of the properties applied to that ID are not necessary here. I added the necessary properties to your style parameter.
```suggestion
                <Whisper preventOverflow trigger="hover" speaker={<Tooltip>Mini Games</Tooltip>}
                     placement="topEnd">
                <IconButton appearance="primary" icon={<Icon icon="gamepad"/>} circle
                            size="lg" onClick={() => Alert.warning("Feature coming soon...")}
                            style={{
                                top: '50%',
                                position: "absolute",
                                right: 5,
                                zIndex: 1
                            }}/>
            </Whisper>
```